go/heif/heif.go

```diff
@@ -600,32 +694,55 @@ func (img *Image) GetImage() (image.Image, error) {
 }
 
 type ImageAccess struct {
-	Plane  []byte
-	Stride int
+	Plane    []byte
+	planePtr unsafe.Pointer
+	Stride   int
+	Height   int
```

Comment:
Why are you not exposing `Width`?

Reply:
Apparently only the height is required (for internal purposes). I will unexport `Height` for now.